### DISCUSSION TITLE
[Question]  isaacsim omni.isaac.sim.python.kit giving error

### DISCUSSION THREAD

**Discussion by meghbhalerao** [isaac-sim/IsaacLab#1391] [Q&A] [answered]
I am trying to install isaac-sim following the docs here - https://isaac-sim.github.io/IsaacLab/main/source/setup/installation/pip_installation.html 

When I run `isaacsim` things work fine and I get an empty window (which I think is expected according to the docs). This window looks like this - 

<img width="1467" alt="Screenshot 2024-10-21 at 12 54 33 PM" src="https://github.com/user-attachments/assets/76d5e19c-f493-4c0f-8d52-0ef51db84707">

Everything works fine up until I run -` isaacsim omni.isaac.sim.python.kit` which gives me the following error - 
```
[Info] [carb] Logging to file: /home/mb230/.nvidia-omniverse/logs/Kit/Isaac-Sim/4.2/kit_20241021_125058.log
2024-10-21 17:50:58 [0ms] [Warning] [omni.kit.app.plugin] No crash reporter present, dumps uploading isn't available.
[0.098s] Failed to solve some dependencies locally, syncing with extension registry...
[0.099s] [ext: omni.kit.async_engine-0.0.1] startup
[0.100s] [ext: omni.client-1.2.0] startup
[0.117s] [ext: omni.kit.registry.nucleus-0.0.0] startup
2024-10-21 17:50:59 [843ms] [Error] [omni.ext.plugin] Failed to resolve extension dependencies. Failure hints:
	[omni.isaac.sim.python-4.2.0] dependency: 'omni.isaac.cortex' = { version='^' } can't be satisfied. Available versions:
	 (none found)
 Synced registries:
	 - kit/default         : found 345 packages 
	 - kit/sdk             : found 320 packages 
	 - kit/community       : found 384 packages 

2024-10-21 17:50:59 [843ms] [Error] [omni.kit.app.plugin] Exiting app because of dependency solver failure...
```

My machine specfications are the following - 

```
Operating System: Ubuntu 24.04.1 LTS              
 Kernel: Linux 6.8.0-45-generic
 Architecture: x86-64
```

I am `ssh-ed` into this machine using `ssh -XY user@host` and `xclock, xeyes, xcalc` work fine.

My local machine is an `M2 MacBook Air`.

**Additional info (may be relevant so including this) -** 
1. I also installed the kit-app-template which is a part of the omniverse from here https://github.com/NVIDIA-Omniverse/kit-app-template (I had done this sort of accidently earlier as I didn't know about the isaac lab and related things)
2. I have the omniverse launcher, and that seems to be working fine (that is if I launch the .AppImage file on my host machine which is the linux box, it renders on my local macbook)

Thanks for the help!

**Comment by Toni-SM** [comment, marked as answer]
Hi @meghbhalerao 

In order to run `isaacsim omni.isaac.sim.python.kit` there is necessary to install all the [Isaac Sim - Python packages](https://docs.omniverse.nvidia.com/isaacsim/latest/installation/install_python.html#install-isaac-sim-using-pip) and not only the few ones Isaac Lab docs indicate (minimal ones to run Isaac Lab).

In any case, your setup if fine. Just skip that step

**Comment by meghbhalerao** [comment]
Thanks for the help! Could this be made clearer in the docs (a possible feature request) here - https://isaac-sim.github.io/IsaacLab/main/source/setup/installation/pip_installation.html#verifying-the-isaac-sim-installation - since the title says **Verifying Isaac installation**, and it could be confusing for the user as they might think that that is an issue with the install.
Please let me know if I am missing something, thanks!

**Comment by chironrong** [comment]
Hello, I encountered the same issue. How did you resolve it?I hope to receive your help.

**Reply by Toni-SM** [reply]
Hi @chironrong 

Please, see https://github.com/isaac-sim/IsaacLab/discussions/1391?converting=1#discussioncomment-11173109
